src/main/java/com/team701/buddymatcher/controllers/timetable/TimetableController.java

```diff
@@ -14,11 +14,10 @@
 import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
 import org.springframework.web.server.ResponseStatusException;
 
-import java.io.ByteArrayInputStream;
-import java.io.IOException;
-import java.io.InputStream;
+import java.io.*;
```

Comment:
It is not a good idea to import more than is required from java.io, and there is an existing case of importing more than is required from org.springframework.web.bind.annotation and java.util that could be refarctored to only include the necessary imports

Reply:
The suggested changes have been made, except for the annotations import as the imports required were 7+ so I think it's better to keep it as is.